contracts/AirbroCampaignFactory.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+import "./campaignAirdrops/NewERC1155DropCampaign.sol";
+import "./campaignAirdrops/ExistingERC20DropCampaign.sol";
+
+/// @title AirbroCampaignFactory - NFT/Token airdrop tool factory contract - for owners of 1155 Nfts
+contract AirbroCampaignFactory {
+    // index of deployed airdrop contracts
+    address[] public airdrops;
+    address public admin = 0xF4b5bFB92dD4E6D529476bCab28A65bb6B32EFb3;
```

Comment:
maybe set in constructor, it gives more flexibility for writing tests and everything

Reply:
Good call, will update